src/Microsoft.ML.StandardLearners/Standard/MultiClass/doc.xml

```diff
@@ -0,0 +1,61 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<doc>
+  <members>
+
+    <member name="MultiClassNaiveBayesTrainer">
+      <summary>
+        Trains a multiclass Naive Bayes predictor that supports binary feature values.
+      </summary>
+      <remarks>
+        <a href ='https://en.wikipedia.org/wiki/Naive_Bayes_classifier'>Naive Bayes</a> is a probabilistic classifier that can be used for multiclass problems.
+        Using Bayes' theorem, the conditional probability for a sample belonging to a class can be calculated based on the sample count for each feature combination groups.
+        However, Naive Bayes Classifier is feasible only if the number of features and the values each feature can take is relatively small.
+        It assumes independence among the presence of features in a class even though they may be dependent on each other.
+        This multi-class trainer accepts binary feature values of type float, i.e., feature values are either true or false.
+        Specifically a feature value greater than zero is treated as true.
+        These learner will request normalization from the data pipeline if the
+        classifier indicates it would benefit from it. Note that even if the
+        classifier indicates that it does not need caching, OVA will always
+        request caching, as it will be performing multiple passes over the data set.
+      </remarks>
+      <seealso cref='Microsoft.ML.Trainers.LogisticRegressionClassifier'/>
+      <seealso cref='Microsoft.ML.Trainers.LightGbmClassifier'/>
+      <seealso cref='Microsoft.ML.Trainers.StochasticDualCoordinateAscentClassifier'/>
+      <seealso cref='Microsoft.ML.Models.OneVersusAll'/>
+    </member>
+    <example name="MultiClassNaiveBayesTrainer">
+      <example>
+        <code language="csharp">
+          pipeline.Add(new NaiveBayesClassifier(){ NormalizeFeatures = NormalizeOption.Auto, Caching = CachingOptions.Memory });
+        </code>
+      </example>
+    </example>
+
+    <member name="OVA">
+      <summary>
```

Comment:
>summary [](start = 7, length = 7)

I think the text below would make a great first paragraph of the remarks section, and summary should be one sentence along the lines of:

Trains a one-versus-all multi-class classifier using a binary classifier as an underlying model.